spec/ttml3.xml

```diff
@@ -1147,6 +1147,16 @@ attribute of a <loc href="#embedded-content-vocabulary-source"><el>source</el></
 content element</loc>.</p>
 </def>
 </gitem>
+<gitem id="terms-external-module">
+<label>[external module]</label>
+<def>
+<p>A module defined externally to this specification in a separate module defining document. An external module
```

Comment:
The specification needs to be considered as the base document plus all of the applicable modules, at least as per the current draft charter proposal at w3c/charter-timed-text#40.

Reply:
In that case, the draft charter needs to be changed.